### PR TITLE
Tiering for GRCh38

### PR DESCRIPTION
**Pull request by jasonwalker80**
Add a UCSC tiering directory for GRCh38 reference sequence build.

### CODE REVIEW

**Comment by susannasiebert** [comment]
Could this be made into a hash instead of an if-else cascade?

**Comment by jasonwalker80** [comment]
new commit with hash suggestion.

**Comment by susannasiebert** [comment]
+1 once tests pass.

File: lib/perl/Genome/Model/Build/ReferenceSequence.pm [hunk 1]
```diff
@@ -191,20 +191,19 @@ sub get{
...
-    }
-    else {
-        $self->status_message("UCSC Tiering Directory not currently available for this species: ".$self->species_name);
+
+    my %tiering_dirs = (
+        '10194788' => '/gscmnt/sata921/info/medseq/make_tier_bed_files/hg18_build36_ucsc_files',
+        '106942997' => '/gscmnt/sata921/info/medseq/make_tier_bed_files/hg18_build36_ucsc_files',
+        '107494762' => '/gscmnt/sata921/info/medseq/make_tier_bed_files/NCBI-mouse-build37/mm9',
+        '3ca0ea4786dd4ebebaf3935f3d3ccab8' => '/gscmnt/sata921/info/medseq/make_tier_bed_files/NCBI-mouse-build38/mm10',
+        '4ec1c5bd1f6941b8a99f2e230217cb91' => '/gscmnt/gc13001/info/model_data/jwalker_scratch/GRCh38',
```
Comment (tmooney) [inline lib/perl/Genome/Model/Build/ReferenceSequence.pm]: Would it make sense to add this to `/gscmnt/sata921/info/medseq/make_tier_bed_files/` like the others?

(It makes me a little sad these are all hard-coded, but that'd be a much bigger change.)

Comment (jasonwalker80) [inline lib/perl/Genome/Model/Build/ReferenceSequence.pm]: I could add them there if I had permissions.  I could easily move them to proposed directory after requesting addition to the medseq group.

Comment (jasonwalker80) [inline lib/perl/Genome/Model/Build/ReferenceSequence.pm]: After requesting the addition to the medseq group, I was made aware that the info_medseq disk groups may be going away.  I pointed out that these data paths would have to be migrated somewhere.  I'm going to leave the files where they are until the location of the existing data is sorted out.

https://jira.gsc.wustl.edu/browse/INFOSYS-18998

**Comment by jasonwalker80** [comment]
@tmooney other suggestions on this PR?

**Comment by tmooney** [comment]
Nope--we'll have to figure out something to do with those files, but we can continue to punt on that. +1.

**Pull request merged**